src/shared/components/ErrorAlert/reducers/errorAlert.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ErrorAlertAction,
+  ErrorAlertAllActions,
+  ErrorAlertState,
+} from '../types/errorAlertTypes';
+
+const errorAlertState = {
```

Comment:
why don't you use the type ErrorAlertState here

Reply:
added